runtime/doc/options.txt

```diff
@@ -3552,6 +3552,31 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  eob		EndOfBuffer		|hl-EndOfBuffer|
 	  lastline	NonText			|hl-NonText|
 
+						*'findexpr'* *'fexpr'*
+'findexpr' 'fexpr'	string	(default "")
+			global or local to buffer |global-local|
+	Expression that is evaluated to obtain the filename for the |:find|
+	command.  When this option is empty, the internal |file-searching|
+	mechanism is used.
+	If a match is found, the expression should return a String containing
+	one or more file names separated by a newline character.  While
```

Comment:
This is probably not robust enough. The expr should return a list of filenames: that way filenames with newlines are handled without issue, and systemlist() becomes easier to use.

Reply:
The PR is updated to support this.

Reply:
Thanks; I don’t have permissions here to resolve my own review comment. Please close this and the memchr comment if possible.